packages/components/Avatar/package.json

```diff
@@ -48,9 +48,12 @@
     "react-native-svg": "^13.14.0"
   },
   "peerDependencies": {
+    "@office-iss/react-native-win32": "^0.72.0",
     "react": "18.2.0",
     "react-native": "^0.72.0",
-    "react-native-svg": "^13.14.0"
+    "react-native-macos": "^0.72.0",
+    "react-native-svg": "^13.14.0",
+    "react-native-windows": "^0.72.0"
```

Comment:
We should make the OOT packages optional with the `peerDependenciesMeta` field. Otherwise, you're telling consumers that they **must** also install them even though they don't care for the platform.

```
  "peerDependenciesMeta": {
    "@office-iss/react-native-win32": {
      "optional": true
    },
    "react-native-macos": {
      "optional": true
    },
    "react-native-windows": {
      "optional": true
    }
  },
```

Reply:
Never mind, it looks like OOT platform types are being used all over the place. This will be difficult to untangle.

Reply:
Most of the typings are centralized to the adapters package. Would you say untangling the types is a prerequisite to getting optional peer dependencies?

Reply:
@acoates-ms is currently looking into it.

Reply:
https://github.com/microsoft/fluentui-react-native/pull/3497 will fix the adapter types to not require the rn-platform packages, so I've switched this back to making the rn-platform packages optional.